### ISSUE TITLE
Use cell structure as a namespace for Cellular Location encoding in Statements.

### ISSUE THREAD

**Issue by abargnesi** [OpenBEL/resource-generator#73]
Currently we have a [CellStructure](http://resource.belframework.org/belframework/20150611/annotation/cell-structure.belanno) annotation and [GOCC](http://resource.belframework.org/belframework/20150611/namespace/go-cellular-component.belns) that pull cell structure terms. With BEL 2.0 we can specify cellular location terms with the `location`, `fromLocation`, and `toLocation` terms.

-----
For existing GOCC namespace:

Incorporate location terms that are children of cellular_component (e.g. [plasma membrane](http://amigo.geneontology.org/amigo/term/GO:0005886)). The new terms should use the Location (L) encoding. These should be able to mix with the existing complex-encodings (C) terms in the same file/ConceptScheme.

For MESH namespace:

Create MESH Cellular structure also as a namespace with Location (L) encodings. These should be equivalenced to GOCC Location terms.

-----

Minimally we can get away with adding Location terms to the GOCC namespace.

@ncatlett Does this make sense?

**Comment by ncatlett** [comment]
Yes. The current Abundance (A) encodings used for location terms in both the GOCC and MESHCS namespaces should be updated with the Location (L) encoding. The existing Complex (C) encodings for GOCC should be maintained.
